### DISCUSSION TITLE
Have single tab closable?

### DISCUSSION THREAD

**Discussion by Norgul** [Sequel-Ace/Sequel-Ace#917] [Ideas]
Hello, 

I think it would be really helpful to have a single DB tab closable as well as multiple tabs are. 

If I want to switch databases now  I need to open a new tab, select DB and close the other one. Alternatively, I need to close entire app and open it to see the DB picker screen again. 

Have the behavior here:
![image](https://user-images.githubusercontent.com/11718157/109382049-550e3780-78de-11eb-8aee-c322052e804b.png)

Same as it is here:
![image](https://user-images.githubusercontent.com/11718157/109382079-7ec75e80-78de-11eb-80ba-edb89a0ef3bc.png)

**Comment by Kaspik** [comment]
Latest beta completely removed custom tabbing and introduced native window tabbing. Check it out or wait for proper release. :)

**Comment by xiangbohua** [comment]
+1

**Comment by Kaspik** [comment]
3.3.0 is out, app now behaves the same as system apps.
